js/request.js

```diff
@@ -150,7 +150,6 @@ exports.request = function(req, res, opts, handler) {
       var terminationMessage = 'Terminating server at http://[localhost]:' + opts.port + ' for ' + opts['module-path'] + ' / ' + opts.handler;
       res.end(terminationMessage + '\n');
       console.info(terminationMessage);
-      server.close();
```

Comment:
re. cleanup: Not that you've got any resources to clean up (just killing the process is fine), but if you did, register a SIGTERM handler to take care of it. Thus, there are now fewer wrong ways to stop the server.